### PR TITLE
[WIP] init tl graph (#394)

### PR DESCRIPTION
**Pull request by lgarithm**
run the following commands to see output

```
make install3
./experimental/tutorial_graph.py
open *.png
```

### CODE REVIEW

File: experimental/tutorial_graph.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+
+import os
+
+import tensorflow as tf
+import tensorlayer as tl
+
+
+def draw_graph(name, net):
+    g = tl.graph.build_graph(net)
```
Comment (zsdonghao) [inline experimental/tutorial_graph.py]: can the graph be a `@property` of the `Layer` ?

**Comment by zsdonghao** [comment]
for mac, to visualize dot file: `brew install graphviz`

**Comment by lgarithm** [comment]
@zsdonghao any master working on this project?

**Comment by DEKHTIARJonathan** [comment]
I just resolved the conflicts. I think we should really focus on it ☺️

**Comment by zsdonghao** [comment]
@lgarithm SiChao Liu is working on that, but the progress is very slow. Douglas and I are worry about that.

**Pull request closed without merging**